src/main/java/browserview/BrowserComponent.java

```diff
@@ -123,12 +130,25 @@ private ChromeDriverEx createChromeDriver() {
         ChromeDriverEx driver = new ChromeDriverEx(options, isTestChromeDriver);
         WebDriver.Options manage = driver.manage();
         if (!isTestChromeDriver) {
+            setWindowSize(manage);
+            initialiseJNA();
+        }
+        return driver;
+    }
+
+    public void setWindowSize(WebDriver.Options manage) {
+        Rectangle availableDimensions = ui.getAvailableDimensions();
+        Rectangle screenDimensions = ui.getDimensions();
+
+        if (availableDimensions.getWidth() > maximiseThreshold * screenDimensions.getWidth()) {
             manage.window().setPosition(new Point((int) ui.getCollapsedX(), 0));
             manage.window().setSize(new Dimension((int) ui.getAvailableDimensions().getWidth(),
                     (int) ui.getAvailableDimensions().getHeight()));
-            initialiseJNA();
+        } else {
+            manage.window().setPosition(new Point(0, 0));
+            manage.window().setSize(new Dimension((int) ui.getDimensions().getWidth(),
```

Comment:
The indentation of the `setSize` part could be improved a bit, to line up the parameters. Helps us see how the two lines are different.